src/main/scala/nl.knaw.dans.easy.deposit/docs/StateInfo.scala

```diff
@@ -27,10 +27,16 @@ case class StateInfo(state: State, stateDescription: String) {
     if (StateInfo.deletableStates.contains(state)) Success(())
     else Failure(new IllegalStateException(s"Deposit has state $state, can only delete deposits with one of the states: ${ StateInfo.deletableStates.mkString(", ") }"))
   }
+
+  def canUpdate: Try[Unit] = {
```

Comment:
Make this name equivalent to the one above.

However, I would suggest `canDelete` there, rather than changing it here. @jo-pol what do you think?